openlayers/index.d.ts

```diff
@@ -11979,8 +11979,8 @@ declare module olx {
          *     decimals: (number|undefined)}}
          */
         interface WriteOptions {
-            dataProjection: ol.ProjectionLike;
-            featureProjection: ol.ProjectionLike;
+            dataProjection?: ol.ProjectionLike;
```

Comment:
This change does not correspond to the documentation. A better modification would be to change the `ProjectionLike` type as Projection | string | null: see [official doc](https://openlayers.org/en/latest/apidoc/ol.html#.ProjectionLike)